aries-site/src/pages/components/tag.mdx

```diff
@@ -0,0 +1,77 @@
+import { Example } from '../../layouts';
+import { TagAnatomy, TagBasicAnatomy } from '../../examples';
+import { FormClose } from 'grommet-icons';
+
+## Guidance
+
+Tags are simple attributes containing metadata:
+
+- Simple, brief and concise metadata to describe a resource or content
+- Included in search, filter, and in the details of a resource
+- Organize and group resources by common characteristics or attibutes
+- Assisst batch actions and selection of multiple resources
+
+## Anatomy
+
+The most basic tags consist of two primary elements:
+
+<Example plain>
+  <TagBasicAnatomy />
+</Example>
+
+1. Border
+1. Name
+
+Tags can have additional elements depending on the tags’ intended use and context:
+
+<Example plain>
+  <TagAnatomy />
+</Example>
+
+1. Border
+1. Name
+1. Value
+1. Close icon
+
+Character requirements:
+
+- Name : 1-128 characters
+- Value : 1-256 characters
+- Unlimited unique names for an account
+
+### The Close Icon
+
+Applying the close icon is dependent upon the use case and not a requirement.
+
+<Example height="xsmall" horizontalLayout>
+  <FormClose />
```

Comment:
I was expecting to see hover/interactive behaviors and the rounding. Execute as a rounded button?